templates/qfdmo/_addresses_partials/adresse_input_form.html

```diff
@@ -3,13 +3,13 @@
     <div data-controller='address-autocomplete'
          data-address-autocomplete-max-option-displayed-value=5
          data-address-autocomplete-is-ban-address-value='true'
-         data-with-controls={{"false" if is_carte(request) else "true"}}
+         data-with-controls={{ is_carte|yesno:"false,true,true" }}
```

Comment:
```suggestion
         data-with-controls="{{ is_carte|lower }}"
```

ce serait pas plus propre ?

Reply:
je ne sais pas :)